types/src/generator/type.ts

```diff
@@ -303,7 +304,11 @@ export function createTypeNode(type: Type, allowCoercion = false): ts.TypeNode {
       }
     case Type_Which.FUNCTION:
       const func = type.getFunction();
-      const params = createParamDeclarationNodes(func.getArgs().toArray());
+      const params = createParamDeclarationNodes(
+        "FUNCTION_TODO",
+        "FUNCTION_TODO",
```

Comment:
This _never_ comes up...thinking about adding a console.log to double check this.